README.md

```diff
@@ -27,7 +27,11 @@ To get a local copy follow these simple steps.
 ```
 npm install
 ```
-4.- To see the home page open index.html with your browser.
+4.- To see the home page run execute below command.
+```
+npm start
+```
+
 
 
 ## Live Demo
```

Comment:
- [ ] (OPTIONAL) Your live demo link for this project is broken. It would be great if you could fix that or remove that section completely in order to make your README file more professional 👍